Libraries/test/Amazon.Lambda.AspNetCoreServer.Test/TestMinimalAPI.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+using Amazon.Lambda.Serialization.SystemTextJson;
+using Amazon.Lambda.APIGatewayEvents;
+using Xunit;
+
+namespace Amazon.Lambda.AspNetCoreServer.Test
+{
+    public class TestMinimalAPI : IClassFixture<TestMinimalAPI.TestMinimalAPIAppFixture>
+    {
+        TestMinimalAPIAppFixture _fixture;
+
+        public TestMinimalAPI(TestMinimalAPI.TestMinimalAPIAppFixture fixture)
+        {
+            this._fixture = fixture;
+        }
+
+        [Fact]
+        public void TestMapPostComplexType()
+        {
+            var response = _fixture.ExecuteRequest<APIGatewayProxyResponse>("minimal-api-post.json");
+            Assert.Equal((int)HttpStatusCode.OK, response.StatusCode);
+            Assert.Contains("works:string", response.Body);
+        }
+
+        public class TestMinimalAPIAppFixture : IDisposable
+        {
+            object lock_process = new object();
+            public TestMinimalAPIAppFixture()
+            {
+            }
+
+            public void Dispose()
+            {
+            }
+
+
+            public T ExecuteRequest<T>(string eventFilePath)
+            {
+                var requestFilePath = Path.Combine(Path.GetDirectoryName(this.GetType().GetTypeInfo().Assembly.Location), eventFilePath);
+                var responseFilePath = Path.GetTempFileName();
+
+                var comamndArgument = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? $"/c" : $"-c";
+                ProcessStartInfo processStartInfo = new ProcessStartInfo();
+                processStartInfo.FileName = GetSystemShell();
+                processStartInfo.Arguments = $"{comamndArgument} dotnet run \"{requestFilePath}\" \"{responseFilePath}\"";
+                processStartInfo.WorkingDirectory = @"C:\codebase\aws-lambda-dotnet\Libraries\test\TestMinimalAPIApp";
```

Comment:
should it be relative path?

Reply:
Opps, I meant to fix that before I committed. It is now fixed.